script/Ciel/Day 14/14_01A_CIEL14A_1.txt

```diff
@@ -221,15 +240,15 @@ But still―――
 [sha:c2ea5c7b39270d15ca8de37ebd91f2f4d0f91998]{
 -- Page 12, Offset 37646.
 -- いいから、頭冷やして出直してこい……！」
-That's enough. Come back after you cool off your head...!"
+That's enough. Come back after you cool your head!"
```

Comment:
I think 出直してこい is more phrasal honestly
Something like this
>Enough——cool your head and then we can talk!

Not sure about いいから, I think it could be a bit more strongly phrased, perhaps? Meaning wise it's kinda close to "Listen here!" but that doesn't quite fit.